### PR TITLE
abinit 7.10.5

### PR DESCRIPTION
**Pull request by mtorrent**
Update abinit to 7.10.5...

### CODE REVIEW

File: abinit.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Abinit < Formula
+  desc "Atomic-scale first-principles simulation software"
   homepage "http://www.abinit.org"
-  url "http://ftp.abinit.org/abinit-7.10.4.tar.gz"
-  sha256 "ebd0a3abd01db4374beda092d1f16c9e00d327712b1ed389bb32e1c80f37c6ef"
-  revision 2
+  url "http://ftp.abinit.org/abinit-7.10.5.tar.gz"
+  sha256 "e9376a3e34790bce90992f28e5fa8554b51ba467bf5709c7fd25d300e7c4f56a"
+  revision 1
```
Comment (dpo) [inline abinit.rb]: You should remove `revision` if you're upgrading.

File: abinit.rb [hunk 2]
```diff
@@ -16,7 +17,7 @@ class Abinit < Formula
 
   depends_on "cmake" => :build
 
-  depends_on :mpi => [:cc, :cxx, :f77, :f90]
+  depends_on :mpi => ["with-fortran", :cc, :cxx, :f77, :f90]
```
Comment (dpo) [inline abinit.rb]: What's `with-fortran`? The `:f77` and `:f90` already take care of that.

**Comment by mtorrent** [comment]
How can we bypass these Travis-CI timeouts?

**Comment by dpo** [comment]
Unfortunately, we can't. Thank you for your contribution!

**Pull request closed without merging**